vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -14,6 +14,8 @@
 from vllm.model_executor.utils import set_weight_attrs
 from vllm.platforms import current_platform
 
+is_hpu = current_platform.is_hpu()
+
```

Comment:
Does this change require vllm-hpu-extension/pull/9 ?
Requirement needs to be bumped in the requirements-hpu.txt

Reply:
right, so I need to bump the file once the extensions PR is merged. and add it to this commit.

Reply:
Thank you, missed the merge somehow :) , updating